core/commands/keystore.go

```diff
@@ -135,6 +142,9 @@ var KeyListCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
 		Tagline: "List all local keypairs",
 	},
+	Options: []cmds.Option{
+		cmds.BoolOption("show-ids", "l", "also show key ids"),
```

Comment:
I would say just `l`, an have it catch all long format output, I think we might have more in future.